aspnetcore/fundamentals/index.md

```diff
@@ -4,17 +4,22 @@ author: rick-anderson
 description: Discover the foundational concepts for building ASP.NET Core apps.
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/20/2018
+ms.date: 10/25/2018
 uid: fundamentals/index
 ---
 # ASP.NET Core fundamentals
 
-An ASP.NET Core app is a console app that creates a web server in its `Main` method:
+An ASP.NET Core app is a console app that creates a web server in its `Program.Main` method. The `Main` method is the app's *managed entry point*:
 
 ::: moniker range=">= aspnetcore-2.0"
 
 [!code-csharp[](index/snapshots/2.x/Program.cs)]
 
+The .NET Core Host, which is usually the [dotnet utility](/dotnet/core/tools/dotnet):
```

Comment:
The word "usually" portrays uncertainty. Can we either do away with that word or explain when the host isn't the dotnet utility?

Reply:
That was one of the reasons that I had the .NET Core Host doc linked. It has that explanation. I'd sort'a kind'a like to not make this too distracting from the overall narrative. We can't drop the word because it's accurate; however, we could drop the whole phrase and just have this line read ...

> The .NET Core Host:

Do you have a preference:

* Add the .NET Core Host link back.
* Drop the end phrase.

I think it would work well either way.

Reply:
I do not think the original .NET Core Host link is appropriate for this document. It explains how to build a custom .NET Core host. It does not explain what the common built-in hosts are.

I would drop the end phrase.

Reply:
Very good ... I'll do that.